app/styles/themes/white-label/_theme.colors.scss

```diff
@@ -0,0 +1,150 @@
+  /* Theme Color Palette */
+
+  /* Neutrals */
+  --color-black:      #{hex2rgb(#000000)};
+  --color-gray-dark:  #{hex2rgb(#333333)};
+  --color-gray:       #{hex2rgb(#afafaf)};
+  --color-light-gray: #{hex2rgb(#f3f3f3)};
+  --color-white:      #{hex2rgb(#ffffff)};
+
+  /* Forms */
+  --color-valid:               #{hex2rgb(#089E00)};
+
+  /* Brand Colors */
+  --color-cool-white:    #{hex2rgb(#EAEFF0)};
+  --color-ceramic-blue:  #{hex2rgb(#235B9C)};
+  --color-concrete:      #{hex2rgb(#8C8C8C)};
+  --color-ember-red:     #{hex2rgb(#C75423)};
+  --color-cool-blue:     #{hex2rgb(#3B8AD1)};
+  --color-sky-blue:      #{hex2rgb(#70C5F6)};
+  --color-sea-green:     #{hex2rgb(#469C78)};
+  --color-bright-purple: #{hex2rgb(#924db1)};
+}
+
+@mixin color-group-light {
+  --color-primary-1:            var(--color-cool-blue);
+  --color-primary-2:            var(--color-sea-green);
+  --color-primary-3:            var(--color-bright-purple);
+  --color-background:           var(--color-white);
+  --color-background-muted:     var(--color-cool-white);
+  --color-background-highlight: var(--color-cool-white);
+  --color-text:                 var(--color-gray-dark);
+  --color-text-muted:           var(--color-concrete);
+  --color-text-error:           var(--color-ember-red);
+  --color-text-inverse:         var(--color-white);
+  --color-text-metadata:        var(--color-gray-dark);
+
+  /* Links */
+  --color-link:                 var(--color-cool-blue);
+  --text-decoration-link:       underline;
+  --color-link-hover:           var(--color-sky-blue);
+  --color-link-hover-background: var(--color-white);
+  --text-decoration-link-hover: underline;
+  --opacity-link-hover:         1;
+
+  /* Navigation Links */
+  --color-navigation-link:                 var(--color-gray-dark);
+  --text-decoration-navigation-link:       underline;
+  --color-navigation-link-hover:           var(--color-sky-blue);
+  --text-decoration-navigation-link-hover: underline;
+  --opacity-navigation-link-hover:         1;
+
+  /* Tags */
+  --color-tag-text:             var(--color-white);
+  --color-tag-background:       var(--color-cool-blue);
+  --color-tag-hover-text:       var(--color-white);
+  --color-tag-hover-background: var(--color-cool-blue);
+  --opacity-tag-hover:          var(--opacity-hover);
+
+  /* Buttons */
+  --color-button-primary:         var(--color-cool-blue);
+  --color-button-primary-hover:   var(--color-cool-blue);
+  --color-button-primary-pressed: var(--color-ceramic-blue);
+  --opacity-button-primary-hover: var(--opacity-hover);
+
+  --color-button-outline:         var(--color-cool-blue);
+  --color-button-outline-hover:   var(--color-cool-blue);
+  --color-button-outline-pressed: var(--color-ceramic-blue);
+  --opacity-button-outline:       var(--opacity-hover);
+
+  /* Borders */
+  --color-border-subtle: var(--color-black);
+  --color-border-standard: var(--color-black);
+  --color-border-bold:   var(--color-black);
+}
+
+@mixin color-group-dark {
```

Comment:
I think i screwed this one up but noticing  a lot of gothamist colors in this dark color group instead of white label colors...

Reply:
I did try and make the same changes on Gothamist and white label, but paid more attention to Gothamist. We'll work on white label after gothamist is on prod!